developers/weaviate/installation/embedded.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Embedded Weaviate (experimental)
+sidebar_position: 4
+# image: TODO
+# tags: ['installation', 'embedded', 'client']
+---
+## Overview
+
+Embedded Weaviate is a new deployment model, which allows you to start a Weaviate instance, straight in your application code using a Weaviate Client library.
+
+:::warning Experimental phase
+Embedded Weaviate is still in the **Experimental** phase.
+
+Some of the APIs and parameters might change over time, as we work towards a perfect implementation.
+:::
+
+### How is this possible?
+
+With every Weaviate [release](https://github.com/weaviate/weaviate/releases) we also publish a single-executalbe Linux binary ([see assets](https://github.com/weaviate/weaviate/releases)).
```

Comment:
typo executable

Reply:
> publish a single-executalbe

what is single supposed to mean here? We also publish ARM64-linux

Reply:
good point
We can change it to:
we also publish executable Linux binaries

would that work?